build/vagrant/infra/policy/s3_upload_policy.json

```diff
@@ -0,0 +1,29 @@
+{
+    "Statement": [
+        {
+            "Action": [
+                "s3:putObject"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:s3:::${artifact_location}/*",
+            "Sid": "AllowS3Upload"
+        },
+        {
+            "Action": [
+                "ssm:GetParameter"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:ssm:us-east-1:701759196663:parameter${ssm_vagrantcloud_token}",
```

Comment:
Template the account ID also. Can use the [aws_caller_identity data source](https://www.terraform.io/docs/providers/aws/d/caller_identity.html) to look it up.

Reply:
May as well also template the [region](https://www.terraform.io/docs/providers/aws/d/region.html).